src/main/resources/templates/header.html

```diff
@@ -1,98 +1,105 @@
 <!doctype html>
 
-<html xmlns:th="http://www.thymeleaf.org">
+<html xmlns:th="http://www.thymeleaf.org" th:fragment="header-info">
```

Comment:
I don't think this is a good idea. Are you going to inject the `<html>` element inside another full HTML document? Doesn't sound right. You should inject the `<header>` element or something like that.

Reply:
Done :)